bower.json

```diff
@@ -35,9 +35,9 @@
   ],
   "dependencies": {
     "firebase": "2.1.x",
-    "ember": "1.9.x",
     "ember-data": "1.0.0-beta.12"
   },
   "devDependencies": {
+    "ember": "1.9.x"
```

Comment:
removing it entirely loaded `1.10.x` and threw an error:

```
Error loading resource file:///Users/tstirrat/dev/emberfire/bower_components/handlebars/handlebars.js (203). Details: Error opening /Users/tstirrat/dev/emberfire/bower_components/handlebars/handlebars.js: No such file or directory
WARNING: Please use `ember.debug.js` instead of `ember.js` for development and debugging.
```

Also, as `ember` updates to the latest stable (e.g. when 1.11.0 comes out, bower will load that), this will continually modify the runtime that the tests run with (could cause more deprecation warnings and errors).

So my options seem to be: 
- fix it at 1.9.x
- fix it at 1.10.x and fix the deprecation and handlebars warnings